docroot/modules/features/resource/config/install/core.entity_view_display.node.resource.default.yml

```diff
@@ -58,9 +58,10 @@ content:
   field_rating:
     weight: 3
     label: above
-    settings: {  }
+    settings:
+      fivestar_widget: modules/fivestar/widgets/basic/basic.css
     third_party_settings: {  }
-    type: fivestar_rating
+    type: fivestar_stars
```

Comment:
@piyuesh23 removed it in https://github.com/NVLI/nvli-main-website/pull/88/commits/6fe67993996517923a46670110dcc6df68842154